### PR TITLE
Mimir 261 form frame rearrange

### PR DESCRIPTION
**Pull request by Glenruben**

### CODE REVIEW

File: src/main/resources/react4xp/_entries/Dashboard/components/RefreshStatisticsForm.jsx [hunk 1]
```diff
@@ -9,7 +13,14 @@ export function RefreshStatisticsForm(props) {
   } = props
 
   const [owners, setOwners] = useState([])
-  const [fetchPublished, setFetchedPublished] = useState(null)
+  const [validated, setValidated] = useState(false)
```
Comment (johnnadeluy) [inline src/main/resources/react4xp/_entries/Dashboard/components/RefreshStatisticsForm.jsx:16]: Lint warning: 'setValidated' is assigned a value but never used  no-unused-vars

File: src/main/resources/react4xp/_entries/Dashboard/components/RefreshStatisticsForm.jsx [hunk 2]
```diff
@@ -23,61 +34,99 @@ export function RefreshStatisticsForm(props) {
...
+    setFetchPublished((prev) => ({
+      ...prev,
+      [ownerObj.ownerId]: newStatus
+    }))
+    updateOwnerCredentials(ownerObj, 'fetchPublished', newStatus)
+  }
+
+  function processForm(event) {
+    event.preventDefault();
+    const form = event.currentTarget;
```
Comment (johnnadeluy) [inline src/main/resources/react4xp/_entries/Dashboard/components/RefreshStatisticsForm.jsx:50]: Lint warning on lines 49 & 50: Extra semicolon

**Review by johnnadeluy — APPROVED** [review]
Just found a few linting nitpicks, otherwise the form looks very good! The border is a nice touch, and everything seems a bit more compact and structured now. 

As for the code, it's very clean and neat as always - easy to grasp. Good job!

File: src/main/resources/react4xp/_entries/Dashboard/components/RefreshStatisticsForm.jsx (hunk 2 above)
Comment (Glenruben) [reply src/main/resources/react4xp/_entries/Dashboard/components/RefreshStatisticsForm.jsx]: Fixed!

File: src/main/resources/react4xp/_entries/Dashboard/components/RefreshStatisticsForm.jsx (hunk 1 above)
Comment (Glenruben) [reply src/main/resources/react4xp/_entries/Dashboard/components/RefreshStatisticsForm.jsx]: Was so sure I had fixed that, but no. Thanks!

**Pull request merged**